UsbAccess/CHANGELOG.md

```diff
@@ -16,6 +16,10 @@ All notable changes to this RDK Service will be documented in this file.
 
 * For more details, refer to [versioning](https://github.com/rdkcentral/rdkservices#versioning) section under Main README.
 
+## [1.0.3] - 2023-03-08
+### Added
+- Added support for 2 more methods in USBAccess plugin
```

Comment:
new APIs require minor version update. Pls refer to https://github.com/rdkcentral/RDKServices#versioning